tools/ClassGen/NodeGen.cpp

```diff
@@ -62,7 +62,7 @@ int main(int argc, char **argv) {
       .addInput("Bias")
       .addMember(MemberType::SizeT, "Kernel")
       .addMember(MemberType::SizeT, "Stride")
-      .addMember(MemberType::SizeT, "Pad")
```

Comment:
can you update the doc string accordingly

Reply:
I updated the one in IR.md from "pad : 2 " to "pads : {2, 2, 2, 2}. The pics need to be updated as well, right?

Reply:
I updated IR.md "pad : 2" to "pads : size_t<2 X 2 X 2 X 2>" .  The pics in docs should be updated as well, right?

Reply:
So the pics in docs should be updated as well, right?

Reply:
that would be good to have.